core/cli/src/init.ts

```diff
@@ -0,0 +1,23 @@
+import { HookClass, Init, InitClass, ValidConfig, Validated, reduceValidated } from '@dotcom-tool-kit/types'
+import { Logger } from 'winston'
+import { importEntryPoint } from './plugin/entry-point'
+
+const loadInitEntrypoints = async (logger: Logger, config: ValidConfig): Promise<Validated<Init[]>> => {
+  const initClassResults = reduceValidated(
+    await Promise.all(
+      config.inits.map(async (entryPoint) => {
+        const hookResult = await importEntryPoint(Init, entryPoint)
+        return hookResult.map((initClass) => initClass as InitClass)
+      })
+    )
```

Comment:
**suggestion**: can be simplified by casting the `importEntryPoint` argument instead of mapping over the `Validated` result
```suggestion
    await Promise.all(config.inits.map((entryPoint) => importEntryPoint(Init as InitClass, entryPoint)))
```